hx_lti_initializer/views.py

```diff
@@ -76,7 +77,13 @@ def initialize_lti_tool_provider(req):
 
 def create_new_user(username, user_id, roles):
     # now create the user and LTIProfile with the above information
-    user = User.objects.create_user(username, user_id)
+    # Max 30 length for person's name, do we want to change this? It's valid for HX but not ATG/FAS
```

Comment:
Christian came up with a (hacky) temporary workaround. Currently we're still using ```lis_person_name_full``` for the username, but whenever there's a duplicate name a space is added to the end to make it unique. Thus the second Test Student becomes ```Test Student ``` (with a trailing space).

It's the same dilemma we were having before. We can't use the HUID, because that gets sent to CATCH alongside the person's name, which would be a security issue. Also, we can't use the hashed version of the HUID because it exceeds the character limit for the model. This solution is far from optimal, but it would only fail if several people have the same long name close to 30 characters, which is highly unlikely. Thoughts?